src/applications/disability-benefits/all-claims/tests/validations.unit.spec.js

```diff
@@ -0,0 +1,48 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+
+import { isValidYear } from '../validations';
+
+describe('526 All Claims validations', () => {
+  describe('isValidYear', () => {
+    it('should add an error if the year is not a number', () => {
+      const err = {
+        addError: sinon.spy(),
+      };
+      isValidYear(err, 'asdf');
+      expect(err.addError.called).to.be.true;
+    });
+
+    it('should add an error if the year contains more than just four digits', () => {
+      const err = {
+        addError: sinon.spy(),
+      };
+      isValidYear(err, '1990asdf');
```

Comment:
I assume this also fails if you test with >4 *digits*, right?

Reply:
If it's > 4 numerical digits, it'll be > 3000 :wink:
So yes. That should be taken care of in another test.